synthetic-monitoring/README.md

```diff
@@ -0,0 +1,44 @@
+# Synthetic Monitoring Jsonnet Library
+
+This library simplifies interaction with the Grafana Synthetic Monitoring API.
+
+Grafana Labs' Synthetic Monitoring is a blackbox monitoring solution provided
+as a part of Grafana Cloud. It provides users with insights into how their
+applications and services are behaving from an external point of view.
+
+It can be configured via an HTTP API. This library allows interaction with
+this API using Jsonnet, making the creation of 'checks' simple.
```

Comment:
Dito, this reads like it actually interacts with the API, but this is not a client.